oembed/async-storage-mock.js

```diff
@@ -0,0 +1,19 @@
+const data = {};
```

Comment:
Create a folder `mocks/`. Move this and `react-native-mock` to that folder, and remove the `-mock` suffix.